Cargo.toml

```diff
@@ -24,6 +24,7 @@ x86-sync-pool = []
 __trybuild = []
 # Enable larger MPMC sizes.
 mpmc_large = []
+defmt-impl = ["defmt"]
```

Comment:
You could remove this line and use `defmt` instead of `defmt-impl` as the feature flag. I think that's that standard way of doing it (e.g. see https://github.com/smoltcp-rs/smoltcp/pull/455).

Reply:
I followed the template provided by ufmt-impl on line 20, if we change this PR we should also change the other in a follow up.